packages/worker/src/worker-options.ts

```diff
@@ -793,15 +849,30 @@ function addDefaultWorkerOptions(options: WorkerOptions, logger: Logger): Worker
     };
   }
 
+  const createPollerBehavior = (defaultMax: number, behavior?: PollerBehavior): Required<PollerBehavior> =>
+    !behavior
+      ? { type: 'simple-maximum', maximum: defaultMax }
+      : behavior.type === 'simple-maximum'
+        ? { type: 'simple-maximum', maximum: behavior.maximum ?? defaultMax }
+        : {
+            type: 'autoscaling',
+            minimum: behavior.minimum ?? 1,
+            initial: behavior.initial ?? 5,
+            maximum: behavior.maximum ?? 100,
+          };
+
+  const wftPollerBehavior = createPollerBehavior(maxWFTPolls, workflowTaskPollerBehavior);
```

Comment:
Is this per WFT poller types? Is there a risk that we might starve either regular or sticky task queue pollers?

Reply:
Also, I assume invalid configuration will be reported by Core?